Dockerfile.sdk

```diff
@@ -209,6 +209,11 @@ COPY qa/L0_client_build_variants qa/L0_client_build_variants
 RUN mkdir -p qa/python_client_unit_tests/ 
 COPY --from=sdk_build /workspace/client/src/python/library/tests/* qa/python_client_unit_tests/
 
+# For CI testing, need to copy over PA docs
```

Comment:
Do we need these in the SDK container? Is this necessary to access them in the QA container?

Reply:
That's a good question, I consider that we need those files for some testing purpose only. 
If it's I can work with @debermudez  to add manage it.